main.tf

```diff
@@ -262,3 +264,35 @@ data "ibm_database_connection" "database_connection" {
   user_id       = ibm_database.elasticsearch.adminuser
   user_type     = "database"
 }
+
+resource "null_resource" "put_vectordb_model" {
+  count = var.enable_elser_model ? 1 : 0
+  provisioner "local-exec" {
+    command     = "${path.module}/scripts/put_vectordb_model.sh"
+    interpreter = ["/bin/bash", "-c"]
+    environment = {
+      HOSTNAME = data.ibm_database_connection.database_connection.https[0].hosts[0].hostname
+      PORT     = data.ibm_database_connection.database_connection.https[0].hosts[0].port
+      PASSWORD = ibm_database.elasticsearch.adminpassword
+    }
+  }
+}
+
+resource "time_sleep" "wait_for_put_trained_model" {
+  depends_on      = [null_resource.put_vectordb_model]
+  create_duration = "300s"
+}
+
+resource "null_resource" "start_vectordb_model" {
+  depends_on = [time_sleep.wait_for_put_trained_model]
+  count      = var.enable_elser_model ? 1 : 0
+  provisioner "local-exec" {
+    command     = "${path.module}/scripts/start_vectordb_model.sh"
+    interpreter = ["/bin/bash", "-c"]
+    environment = {
```

Comment:
can we pass created ES constructed endpoint directly, in that case we pass only one parameter

Reply:
done